docs/usage/evaluation_examples.md

```diff
@@ -1,33 +1,6 @@
 # Evaluation Examples Using curl
 
-## Setup
```

Comment:
Why was this section removed?

Reply:
All commands are now available in the main readme and it makes this section redundant.

Reply:
I think we'll need it when we incorporate these docs into the main doc page because it likely won't include the main readme. It's a bit redundant but I think that's okay in this case.

Reply:
Added this back in. Resolving conversation.